### ISSUE TITLE
What was the last version to build with Qt4?

### ISSUE THREAD

**Issue by barracuda156** [filecxx/FileCentipede#699]
Apparently the current version of `FileCentipede` forces Qt5. Could someone refer me to the latest one which built with Qt4?

**Comment by filecxx** [comment]
FileCentipede 1.0 was built with Qt 5.15

**Comment by barracuda156** [comment]
@filecxx Oh…

And it does rely on stuff which does not exist with Qt4, right? I mean, I am gonna build from source, not use pre-compiled binaries, but re-writing Qt implementation is not realistic.

**Comment by barracuda156** [comment]
@filecxx By the way, can it be built without GUI, as a CLI client?

**Comment by filecxx** [comment]
No, but you will have the CLI client after FileCentipede3.0 is released.

**Comment by barracuda156** [comment]
Thank you, that will be great. Anything `ncurses`-based or pure CLI should work for our case.
